tmule/loader.py

```diff
@@ -14,13 +14,13 @@ def __init__(self, stream):
     def include(self, node):
 
         data = []
-        for file in str(self.construct_scalar(node)).split(' '):
+        for file in str(self.construct_scalar(node)).split(';'):
             if (file[:1] == '$'):
-                filename = os.path.expandvars(file)
+                filename = os.path.expandvars(os.popen("echo " + file).read()[:-1])
```

Comment:
Looks a bit hacks, but should work I guess. How are errors handled and reported, i.e., when I have a typo in my command and it is not an actual shell executable command?

Reply:
Errors are not handled explicitly, an exception is thrown if the filepath is not available. So this is no different to the current implementation.

`IOError: [Errno 2] No such file or directory: '/path/to/file.yaml'` 

So it will inform you of the specific import which was the problem-causer, but not how it caused the problem. There may be better ways to do this, but `echo path` seems stable.

Reply:
Ok, error handling is fine, but changing the example doesn’t address the real problem, does it?

Reply:
I thought it appropriate to update the example to match the changed use of `!include`, that way the CI test can properly evaluate if there are any issues with it. Given the CI test failed on the example:

`FileNotFoundError: [Errno 2] No such file or directory: 'robot.yaml robot_extras.yaml'`

I think once a decision and implementation is decided for versioning, it would make sense to include an example of each version.

Reply:
I agree @marc-hanheide , stating the version in the config file, with default version 1 if not specified, seems the best solution here.